docs/user-guide/secrets/index.md

```diff
@@ -234,6 +280,11 @@ $ cat /etc/foo/password
 The program in a container is responsible for reading the secret(s) from the
 files.
 
+**Mounted Secrets are updated automatically**
```

Comment:
Use double-hash (which notes headings in Markdown) instead of surrounding in asterisks